commons/commons-util/src/main/kotlin/ebi/ac/uk/util/collections/CollectionExtensions.kt

```diff
@@ -37,9 +35,9 @@ fun <T> List<T>.second() =
 fun <T> List<T>.third() =
     if (this.size > 2) this[2] else throw NoSuchElementException("List does not contain a third element.")
 
-fun <T> List<T>.findSecond() = if (this.size > 1) Option.just(this[1]) else Option.empty()
+fun <T> List<T>.findSecond() = if (this.size > 1) this[1] else null
```

Comment:
Should we add the return type? I think we defined we were setting the return type for all public methods